docker-compose.yml

```diff
@@ -56,8 +56,12 @@ services:
       - KEYCLOAK_HOST=${KEYCLOAK_HOST}
       - KEYCLOAK_REALM=${KEYCLOAK_REALM}
       - KEYCLOAK_CLIENT_ID=${KEYCLOAK_CLIENT_ID}
+      - KEYCLOAK_INTEGRATION_ID=${KEYCLOAK_INTEGRATION_ID}
+      - KEYCLOAK_ENVIRONMENT=${KEYCLOAK_ENVIRONMENT}
+      - KEYCLOAK_ADMIN_HOST=${KEYCLOAK_ADMIN_HOST}
       - KEYCLOAK_ADMIN_USERNAME=${KEYCLOAK_ADMIN_USERNAME}
       - KEYCLOAK_ADMIN_PASSWORD=${KEYCLOAK_ADMIN_PASSWORD}
+      - KEYCLOAK_API_HOST=${KEYCLOAK_API_HOST}
```

Comment:
We'll need to double check that these new values are being passed through the openshfit pipeline files as well